code/game/objects/items/weapons/flame.dm

```diff
@@ -5,11 +5,6 @@
 	var/lit = FALSE
 	material = /decl/material/solid/organic/wood
 
-/obj/item/flame/afterattack(var/obj/O, var/mob/user, proximity)
```

Comment:
Is this handled somewhere else now?

Reply:
I see, item-level. I hope this won't fuck with things like igniting rags on bottles or lighting cigarettes.